puregdb/puregdb.py

```diff
@@ -288,7 +288,7 @@ def userfree(self):
 
             entrySize = int(entry['xBlockSize']) & 0x7fffffff
             address=int(entry.address)
-            print("userHeap block addr: ", address, "\t\tsize: ", entrySize)   
+            # print("userHeap block addr: ", address, "\t\tsize: ", entrySize)   
```

Comment:
Why commented out, not thrown out?

Reply:
because if something is wrong with the system heap then these logs will be very helpful :)